Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -7,6 +7,10 @@
 using LootLocker.LootLockerEnums;
 using static LootLocker.LootLockerConfig;
 using System.Linq;
+using static System.Net.WebRequestMethods;
+using static System.Runtime.CompilerServices.RuntimeHelpers;
+using System.Diagnostics;
+using File = System.IO.File;
```

Comment:
Are these intentionally added?

Reply:
Yeah, for some reason it didn't compile without it O.o